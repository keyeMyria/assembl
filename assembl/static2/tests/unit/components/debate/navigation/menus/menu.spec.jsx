import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Menu from '../../../../../../js/app/components/debate/navigation/menus/menu';

describe('MenuTable component', () => {
  it('should match the survey table', () => {
    const props = {
      identifier: 'survey'
    };
    const renderer = new ShallowRenderer();
    renderer.render(<Menu {...props} />);
    const rendered = renderer.getRenderOutput();
    expect(rendered).toMatchSnapshot();
  });

  it('should match the default table (ideas table)', () => {
    const props = {
      identifier: 'foo'
    };
    const renderer = new ShallowRenderer();
    renderer.render(<Menu {...props} />);
    const rendered = renderer.getRenderOutput();
    expect(rendered).toMatchSnapshot();
  });
});