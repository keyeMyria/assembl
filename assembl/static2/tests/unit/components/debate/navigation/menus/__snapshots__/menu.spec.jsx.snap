// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuTable component should match the default table (ideas table) 1`] = `
<Connect(Apollo(Component))
  identifier="foo"
/>
`;

exports[`MenuTable component should match the survey table 1`] = `
<Connect(Apollo(Component))
  identifier="survey"
/>
`;
